apps/ledger-live-desktop/src/renderer/modals/SettingsAccount/AccountSettingRenderBody.tsx

```diff
@@ -253,12 +253,12 @@ class AccountSettingRenderBody extends PureComponent<Props, State> {
               danger
               type="button"
               onClick={this.handleOpenRemoveAccountModal}
-              id="account-settings-delete"
+              data-test-id="account-settings-delete-button"
```

Comment:
Just double checking nothing used this as an `id` before?

Reply:
Nope 👌 thank you I checked again